### PR TITLE
feat(): add ux improvements

### PR DESCRIPTION
**Pull request by lucbpz**
Address all UX improvements from our design team.

### CODE REVIEW

**Review by felipecruz91 — COMMENTED** [review]
After emptying the volume, the size needs to be recalculated (or set it to 0 directly if the empty action was successful).

**Review by gtardif — APPROVED** [review]
LGTM appart from @felipecruz91 comment on empty volume action (well spotted! )

File: ui/src/App.tsx [hunk 1]
```diff
@@ -510,13 +480,25 @@ export function App() {
...
+              </Button>
+            </Grid>
+            <DataGridPro
               loading={isLoading}
               components={{
                 LoadingOverlay: LinearProgress,
                 Toolbar: () => (
                   <CustomToolbar
-                    openDialog={() => setOpenImportIntoNewDialog(true)}
+                  // openDialog={() => setOpenImportIntoNewDialog(true)}
```
[the remark this answers is not in the document]
Comment (lucbpz) [reply ui/src/App.tsx]: done!

**Comment by felipecruz91** [comment]
Added the following request to this PR too:

- https://github.com/docker/volumes-backup-extension/issues/67

**Review by gtardif — CHANGES_REQUESTED** [review]
Need to fix MUI license watermark. This can be tested by installing the extension build for this PR in the CI, it should normally have the secret license key

File: .github/workflows/build-scan-push.yaml [hunk 2]
```diff
@@ -91,14 +91,15 @@ jobs:
           labels: |
             org.opencontainers.image.revision=${{ github.event.pull_request.head.sha || github.event.after || github.event.release.tag_name }}
             org.opencontainers.image.source=https://github.com/${{ github.repository }}
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
+#          cache-from: type=gha
+#          cache-to: type=gha,mode=max
```
Comment (felipecruz91) [inline .github/workflows/build-scan-push.yaml:95]: @gtardif The issue with the license key was that the stage was cached, now it works fine:

image

Comment (felipecruz91) [inline .github/workflows/build-scan-push.yaml:95]: Undo:

```suggestion
#          cache-from: type=gha
#          cache-to: type=gha,mode=max
```

**Comment by felipecruz91** [comment]
@gtardif LGTM

image

**Pull request merged**